src/markdown-to-html-parser.js

```diff
@@ -2,9 +2,9 @@ const showdown = require('showdown')
 
 // there may be an option to enable this, but since we haven't found it here is a reg-ex
 // to convert links within a file from *.md to *.html
-const linkExtensionExtension = {
+const convertMdLinksToHtmlLinks = {
   type: 'output',
-  regex: /<a href="(.*).md">/g,
+  regex: /<a href="([^:]*).md">/g, // exclude colon, so external links aren't converted
```

Comment:
This will now link to GitHub even if the link is actually contained within the repository, but not relatively linked, but I think that is reasonable.

I.e. I have a file located in '../file.md', but I link to it like `[file](https://github.com/myrepo/file.md)`